tools.go

```diff
@@ -5,6 +5,7 @@ package main
 import (
```

Comment:
why is this file needed?

Reply:
This file is here to track versions of the tools that we use for the project, with new go modules. in this context, a tool is binary that is used at build time or development time. For further information, check: https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module